lib/Button.js

```diff
@@ -17,7 +18,10 @@ module.exports = React.createClass({
   },
 
   getDefaultProps: function() {
-    return { tag: 'span' };
+    return {
+      disabled: false,
```

Comment:
I'm fine with adding disabled here, though it's unrelated to the PR, right? 

If you do that, you should change its propType to `isRequired`, because it's always there.